Validate README instructions (#2)
cloudflare/odoh-client-go: README.md

> # odoh-client-go

Author: Somewhat opinionated: my feeling is that the README title should match the package name. I'll revert if you disagree.

`odoh-client` is a command line interface as a client for making [Oblivious DNS-over-HTTPS](https://tools.ietf.org/html/draft-pauly-dprive-oblivious-doh-03) queries. The implementation is based on [chris-wood/odoh-client](https://github.com/chris-wood/odoh-client).

It currently supports the following functionalities:

> - [x] DoH Query: `odoh-client doh --domain www.cloudflare.com. --dnstype AAAA --target <target>` where `<target>` is the XXX, e.g., `odoh.cloudflare-dns.com`.

Author: Replace `XXX` with a description of the target.

> - [x] DoH Query: `odoh-client doh --domain www.cloudflare.com. --dnstype AAAA --target <target>` where `<target>` is the XXX, e.g., `odoh.cloudflare-dns.com`.

Reviewer: ```suggestion
- [x] DoH Query: `odoh-client doh --domain www.cloudflare.com. --dnstype AAAA --target <target>` where `<target>` is the name of the target resolver, e.g., `odoh.cloudflare-dns.com`.
```

- [x] ODoH Query: `odoh-client odoh --domain www.cloudflare.com. --dnstype AAAA --target <target>`

> - [x] ODoH Query via Proxy: `odoh-client odoh --domain www.cloudflare.com. --dnstype AAAA --target <target> --proxy <proxy>` where `<proxy>` is the XXX, e.g., `odoh1.surfdomeinen.nl`.

Author: Replace XXX with a description of the proxy.

> - [x] ODoH Query via Proxy: `odoh-client odoh --domain www.cloudflare.com. --dnstype AAAA --target <target> --proxy <proxy>` where `<proxy>` is the XXX, e.g., `odoh1.surfdomeinen.nl`.

Reviewer: ```suggestion
- [x] ODoH Query via Proxy: `odoh-client odoh --domain www.cloudflare.com. --dnstype AAAA --target <target> --proxy <proxy>` where `<proxy>` is the name of a proxy server.
```
